plugins/Morpheus/templates/_jsGlobalVariables.twig

```diff
@@ -16,14 +16,13 @@
         symbolDecimal: "{{ 'Intl_NumberSymbolDecimal'|translate }}"
     };
 
+    piwik.relativePluginWebDirs = {{ relativePluginWebDirs|json_encode|raw }}
+
     {% if userLogin %}piwik.userLogin = "{{ userLogin|e('js')}}";{% endif %}
 
     {% if idSite is defined %}piwik.idSite = "{{ idSite }}";{% endif %}
 
-    {% if siteName is defined %}
-    // NOTE: siteName is currently considered deprecated, use piwik.currentSiteName instead, which will not contain HTML entities
-    piwik.siteName = "{{ siteName|e('js') }}";
-    piwik.currentSiteName = {{ siteNameDecoded|json_encode|raw }};{% endif %}
+    {% if siteName is defined %}piwik.siteName = "{{ siteName|e('js') }}";{% endif %}
```

Comment:
I think the code deleted on the left shouldn't be deleted, right?

Reply:
Indeed that should not be removed. Was probably due to some later merge. Will fix that

Reply:
fixed @diosmosis